script.js

```diff
@@ -0,0 +1,97 @@
+var script = document.createElement('script');
+script.src = "https://checkout.razorpay.com/v1/checkout.js";
+
+window.onload  = function(e)
+{
+    document.body.appendChild(script);
+
+    script.onload = function()
+    {
+        var script2  = document.createElement("script");
+
+        var div = document.createElement('div');
+
+        var html = "<form id ='razorpayform' name='razorpayform' method='POST'> " +
+        "<input type='hidden' name='razorpay_payment_id' id='razorpay_payment_id'> " +
+        "<input type='hidden' name='razorpay_signature'  id='razorpay_signature' > " +
```

Comment:
@riagarg5 Please use heredocs. Please refer to quick payments / woocommerce to see how we do use heredocs.